howso/client/__init__.py

```diff
@@ -16,26 +16,26 @@
 Examples implementations are included in the howso/examples directory.
 """
 
-from .base import AbstractHowsoClient  # noqa: F401
```

Comment:
Changes to imports in this file seem uncessary.

Reply:
Ths was residual from when I was trying to solve the cyclic import issues when doing multi-worker testing.

Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60

Reply:
Doesn't look like any further changes were made?